src/bin/cli/logs/compare.rs

```diff
@@ -0,0 +1,150 @@
+use catalyst_toolbox::logs::sentry;
+use catalyst_toolbox::logs::sentry::{LogCmpStats, RawLog, SentryFragmentLog};
+use chain_core::property::Fragment;
+use jcli_lib::utils::io;
+use jormungandr_lib::interfaces::{
+    load_persistent_fragments_logs_from_folder_path, PersistentFragmentLog,
+};
+use serde::de::DeserializeOwned;
+use std::path::PathBuf;
+use structopt::StructOpt;
+use url::Url;
+
+#[derive(thiserror::Error, Debug)]
+pub enum Error {
+    #[error("api-token parameter must be provided if sentry-url is set")]
+    MissingTokenParameter,
+
+    #[error(transparent)]
+    SentryLogsError(#[from] sentry::Error),
+
+    #[error(transparent)]
+    IoError(#[from] std::io::Error),
+
+    #[error("error deserializing logs from: {path}")]
+    DeserializeError {
+        path: PathBuf,
+        source: serde_json::Error,
+    },
+}
+
+#[derive(StructOpt)]
+#[structopt(rename_all = "kebab-case")]
+pub struct Compare {
+    #[structopt(long, conflicts_with = "sentry-logs")]
+    sentry_url: Option<Url>,
+
+    #[structopt(long, conflicts_with = "sentry-logs")]
```

Comment:
Could this conflict be handled with an enum and a `structopt(flatten)`?

Reply:
IIRC `conflicts_with` didn't work well with flatten attributes.  It happen before. But I'll double check.

Reply:
I removed this option, it is better to use downloaded logs than directly use them, since it may be a bit slow.